Libraries/LinkingIOS/RCTLinkingManager.mm

```diff
@@ -17,6 +17,21 @@
 
 static NSString *const kOpenURLNotification = @"RCTOpenURLNotification";
 
+#if TARGET_OS_OSX // TODO: https://github.com/microsoft/react-native-macos/issues/632
```

Comment:
Not totally sure why this shows as a diff but changes to RCTLinkingManager, podfile.lock, and React-RCTLinking.podspec are in the build in Devmain and can be ignored here. We want these changes as it reflects the [last commit](https://github.com/microsoft/react-native-macos/commit/efb8132db494bec4a2325e68749cd02ffcc1be1a) that went in to devmain.

Reply:
Is the head of your branch not correct? Try creating a new clone, pulling from later 0.62 stable and see if you see the same issue?